### DISCUSSION TITLE
Uno Default Template: back icon not showing when published

### DISCUSSION THREAD

**Discussion by DrHardReset** [unoplatform/uno#14963] [Q&A]
I just created a new instance of the default UNO template in VS.
When building and running the application the back button on "secondPage" is showing.
When publishing and running the app, the icon of the back button is not showing. (clicking is possible though).

publish settings:
![grafik](https://github.com/unoplatform/uno/assets/35699007/eb39f1f8-d583-4e87-9c2e-f3e03871dcbe)

![grafik](https://github.com/unoplatform/uno/assets/35699007/2caef907-fd39-4215-a6f5-80fb38bd0013)

Is this a bug or am I missing something?

**Comment by jeromelaban** [comment]
@kazo0 does this look familiar?

**Reply by kazo0** [reply]
I haven't seen this happening only when in a published app. Perhaps Resizetizer is somehow not packaging the generated back.png properly?

**Reply by jeromelaban** [reply]
Indeed. @DrHardReset do you see the back.png file in your app's output folder?

**Reply by DrHardReset** [reply]
The folder \<ProjectTitle\> (in my case "UnoApp1") which contains the Assets is missing within the published folder.

build:
![build](https://github.com/unoplatform/uno/assets/35699007/468946b1-344c-4bb9-9edf-c0db572d241c)



publish:
![publish](https://github.com/unoplatform/uno/assets/35699007/f872675d-9e03-4542-b9d1-d5926a42ea09)


When I manually copy the missing folder to the published folder and run the exe, the back button is displayed.

**Reply by jeromelaban** [reply]
Interesting. I wonder if it's a packaged vs. unpackaged issue on the WinAppSDK's side.
